doc/build/dialects/index.rst

```diff
@@ -88,7 +88,10 @@ Currently maintained external dialect projects for SQLAlchemy include:
 +---------------------------------------+---------------------------------------+
 | Teradata Vantage                      | teradatasqlalchemy_                   |
 +---------------------------------------+---------------------------------------+
+| IBM Netezza Performance Server        | nzalchemy_                            |
```

Comment:
I think the table is currently in alphabetic order, so it would be nice if you could move this after `IBM DB2 and Informix `

Reply:
Thanks for pointing out. I have put that in alphabetical order now. 
And yes, we currently support 1.3 so it should go to 1.3 documentation. That is why I have forked my branch from rel_1_3 and targeted the PR to rel_1_3. Please let me know if I am missing something.